File: EnglishTutorAI/ClientApp/src/app/modules/user/models/story.ts
import { StoryParagraph } from "./story-paragraph";

export class Story {
    storyId: number = 0;
    creationDate: Date = new Date();
    modificationDate: Date | null = null;
    status: number = 0;
    title: string = "";
    paragraphs: StoryParagraph[] = [];
}

File: DbModels/Story.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace DbModels;

public class Story
{
	[Key]
	public int StoryId { get; set; }
	[Required]
	public DateTime CreationDate { get; set; }
	public DateTime? ModificationDate { get; set; }
	[Required]
	[DefaultValue(StoryStatus.Unread)]
	public StoryStatus Status { get; set; }
	[Required]
	[MaxLength(500)]
	public string Title { get; set; } = string.Empty;
	public ICollection<StoryParagraph> Paragraphs { get; set; }
    public string UserId { get; set; }
	public ApplicationUser User { get; set; }
}

public enum StoryStatus
{
	Unread,
	Done
}